bin/common/write_k8s_ip_to_environment.sh

```diff
@@ -0,0 +1,8 @@
+#!/usr/bin/env bash
+
+set -o errexit
+while ! ifconfig eth1 | grep -q addr; do
+  sleep 3
+done
+K8S_VM_IP=$(ifconfig eth1 | grep -oE 'addr:[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}' | sed 's/addr://g')
+echo K8S_VM_IP=$K8S_VM_IP >> /etc/environment
```

Comment:
Tested cert generation without any special IPs and everything seems to work... so I should maybe just shove everything back into install_kubernetes and call it from the *packer* provisioner again